tony-history-server/app/controllers/JobEventPageController.java

```diff
@@ -0,0 +1,60 @@
+package controllers;
+
+import cache.CacheWrapper;
+import com.google.common.base.Preconditions;
+import com.google.common.cache.Cache;
+import com.linkedin.tony.TonyConfigurationKeys;
+import com.typesafe.config.Config;
+import hadoop.Configuration;
+import java.util.List;
+import javax.inject.Inject;
+import models.JobEvent;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.hdfs.HdfsConfiguration;
+import play.Logger;
+import play.Logger.ALogger;
+import play.mvc.Controller;
+import play.mvc.Result;
+import utils.HdfsUtils;
+
+import static utils.HdfsUtils.*;
+import static utils.ParserUtils.*;
+
+
+public class JobEventPageController extends Controller {
+  private static final ALogger LOG = Logger.of(JobEventPageController.class);
```

Comment:
ALogger is a funny class name. I wonder why Play named it that as opposed to just Logger.